test/plugins/data/build.json

```diff
@@ -9,6 +9,9 @@
     "endTime": "2038-01-19T03:16:10.131Z",
     "parameters": {},
     "stats": {},
+    "environment": {
```

Comment:
can we have a test case with multiple values ?

Reply:
actually added this here to only show we have `environment` in build object, but the test cannot cover the converting because the `update` function is from model and it's mocked.